test/lightning_web/end_to_end_test.exs

```diff
@@ -188,4 +293,38 @@ defmodule LightningWeb.EndToEndTest do
       return state;
     });"
   end
+
+  defp start_runtime_manager(_context) do
+    rtm_args = "node ./node_modules/.bin/worker -- --backoff 0.5/5"
+
+    Application.put_env(:lightning, RuntimeManager,
+      start: true,
+      args: String.split(rtm_args),
+      cd: Path.expand("../../assets", __DIR__)
+    )
+
+    {:ok, rtm_server} = RuntimeManager.start_link(name: E2ETestRuntimeManager)
```

Comment:
I think we can pass those options directly into `start_link/1`, the start args are merged into the Application env - if we do that and we want to start the worker elsewhere we won't have competing application envs.

Reply:
true, previously the other args were ignored with _args.

Reply:
has not worked though [cdf4773](https://github.com/OpenFn/Lightning/pull/1370/commits/cdf4773d0473cb23c2adb95f41db34d462624215) 
reverted on [c476be9](https://github.com/OpenFn/Lightning/pull/1370/commits/c476be939ea04ce95a54cae49fb91a96eeed55b8)